src/sections/Home/So-Special-Section/so-special-style.js

```diff
@@ -93,70 +93,59 @@ const SoSpecialWrapper = styled.div`
         background-color:white;
         color:black;
     }
-
-    
+    .so-special-foot{
+        text-align:center;
+        .so-special-foot-btn
+        {
+            color:white;
+            margin:2rem;
+            :hover{
```

Comment:
Hover effect is set for the button component

Reply:
Yes I agree . But I think color of the text inside it should change to black on hover . If I remove it ,text color is not changing ,that's why I have set it up separately.